src/app/shared/enum/enumUA/message-bar.ts

```
@@ -112,5 +115,17 @@ export enum SnackbarText {
   deleteDraftFail = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DELETE_DRAFT_FAIL',
   deleteDraftSuccess = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DELETE_DRAFT_SUCCESS',
   rejectDraftSuccess = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DRAFT_REJECTED_SUCCESS',
-  approveDraftSuccess = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DRAFT_APPROVED_SUCCESS'
+  approveDraftSuccess = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DRAFT_APPROVED_SUCCESS',
+
+  workshopCoverImageDeleted = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.COVER_IMAGE_DELETED',
+  workshopImageDeleted = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.IMAGE_DELETED',
+  editDraft = 'SERVICE_MESSAGES.SNACK_BAR_TEXT.DRAFT_EDITED'
+}
+
+export function showHttpErrorMessage(dispatch: Function, status?: number, defaultError: SnackbarText = SnackbarText.error): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)